.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+on:
+  workflow_call:
+    inputs:
+      cli-name:
+        required: true
+        type: string
+name: Release
+jobs:
+  release:
+    permissions:
+      id-token: write
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+      - name: Run GoReleaser
+        uses: goreleaser/goreleaser-action@v2
+        with:
+          distribution: goreleaser
+          version: "v1.6.3"
```

Comment:
Apparently, goreleaser releases frequently, because when I opened the PR 1.6.3 was the latest one. I'm open to suggestions 👍 

Latest is the default, but I'd suggest pinning to a specific version rather than using "latest" as it improves the consistency of the CI behavior.